projects/replicated-ship/schema.json

```diff
@@ -75,29 +74,98 @@
                   },
                   "dest": {
+                    "description": "The directory in which the layer should be unpacked",
+                    "type": "string"
+                  },
+                  "image": {
+                    "description": "The docker image URL",
+                    "type": "string"
+                  },
+                  "layer": {
+                    "description": "a SHA256 of the layer to extract. You can inspect the layer SHAs of a docker image by running something like `docker save myimage > myimage.tar && tar xvf myimage.tar -C /tmp`. Ship will unpack the archive at `<layer sha>/layer.tar` to the `dest` directory.",
                     "type": "string"
                   },
                   "mode": {
+                    "description": "The unix file permissions to be set on target unpack directory, e.g `777`",
                     "type": "integer"
                   },
-                  "path": {
+                  "source": {
+                    "description": "The source for the image. Should be either `public`, `replicated`, or the name of a third-party private registry previously configured on [console.replicated.com](https://console.replicated.com)",
                     "type": "string"
-                  },
-                  "ref": {
+                  }
+                },
+                "required": [
+                  "dest",
+                  "image",
+                  "source",
+                  "layer"
+                ]
+              },
+              "helm": {
+                "description": "A `helm` asset will template and render an existing helm chart. Your installer runbook should included assets to drop an existing helm chart at `local.chart_root`. Future releases will include support for charts stored in private github repos or in chart repositories like ChartMuseum.",
+                "examples": [
+                  {
+                    "values": {
+                      "AppFlavor": "ship",
+                      "NginxReplicas": "{{repl ConfigOption \\\"nginx_replicas\\\"}}"
+                    },
+                    "helm_opts": [
+                      "--name",
+                      "{{repl Installation \\\"channel_name\\\"}}-{{repl Installation \\\"semver\\\"}}"
+                    ],
+                    "local": {
+                      "chart_root": "charts/src/nginx/"
+                    },
+                    "dest": "charts/rendered/"
+                  }
+                ],
+                "type": "object",
+                "properties": {
+                  "description": {
                     "type": "string"
                   },
-                  "repo": {
+                  "dest": {
+                    "description": "The directory in which to render the. If the source chart is at `charts/src/nginx`, and `dest` is set to `charts/rendered/`, then the chart will be templated at `charts/rendered/nginx`",
```

Comment:
`The directory in which to render the.` 

This is probably missing something

Reply:
i swear I fixed this cmon github https://github.com/replicatedhq/ship/pull/115#discussion_r196141659